client/src/js/components/bhBreadcrumb.js

```diff
@@ -95,9 +95,12 @@ function BreadcrumbController() {
    */
   vm.bcDropdowns = vm.dropdown || [];
 
-  /** call the appropriate function and update the dropdown label */
+  vm.dropDownItemSelectedLabel = ''; //for the test purpose
```

Comment:
This function is difficult to test, it is just a way to make it testable, I have tried an other way but no success fill free to help me if you have a good idea.